package.json

```diff
@@ -5,7 +5,13 @@
   "license": "Apache-2.0",
   "private": true,
   "scripts": {
-    "build": "webpack --mode production && node scripts/copyNodeModules.js",
+    "build": "yarn build:client && yarn build:server",
+    "watch": "yarn watch:client & yarn watch:server",
+    "build:client": "webpack --mode production && node scripts/copyNodeModules.js",
+    "build:client:dev": "webpack --config webpack.development.js && node scripts/copyNodeModules.js",
+    "watch:client": "webpack --watch --config webpack.development.js",
+    "build:server": "node esbuild.server.js",
+    "watch:server": "WATCH=1 yarn build:server & nodemon --inspect build/server.js",
```

Comment:
Is there a use case for these yarn scripts? (It's possible there is and I'm not seeing it) I find it's better to keep the list of scripts in the package file as small as possible so that new devs have a clearer idea of what commands they _should_ run.

My thinking is, if I shouldn't run this command as part of my dev routine, then it shouldn't be here.

Reply:
Yeah, I agree. I think these can be just in-lined into the `dev` and `build` commands. I can't think of a reason as to when we'd have to run one of these commands separately

Reply:
got a bit carried away with the scripts there, I'll narrow these down to dev and build as suggested